starter-ui/mvnw

```diff
@@ -0,0 +1,316 @@
+#!/bin/sh
```

Comment:
Do we really need this? This would really be just for committers and contributors. I think we should assume such people can easily set up Maven.

Reply:
True. 
I wanted to make it easy for Jeff.  If other UI developers join us, they wouldn't need to install maven to run the project to checkout.